component-library/component-lib/src/lib/language-switch/language-switch.component.ts

```diff
@@ -29,14 +29,14 @@ export class LanguageSwitchComponent implements OnInit {
     @Inject(PLATFORM_ID) private platformId: object,
     private langToggle: LanguageSwitchButtonService,
     private translate: TranslateService) { 
-      this.isMobile = window.innerWidth <= 360;
+      this.isMobile = window.innerWidth <= 768;
```

Comment:
Please undo the changes here - mobile should be set at 360px

Reply:
Unless I've missed something

Reply:
breakpoints were changed to tablet breakpoints after checking with sara from design.